src/NuGet.Core/NuGet.Configuration/PublicAPI.Unshipped.txt

```diff
@@ -1,28 +1,28 @@
-NuGet.Configuration.NamespaceItem
-NuGet.Configuration.NamespaceItem.Id.get -> string
-NuGet.Configuration.NamespaceItem.NamespaceItem(string id) -> void
-NuGet.Configuration.PackageNamespacesConfiguration
-NuGet.Configuration.PackageNamespacesConfiguration.AreNamespacesEnabled.get -> bool
-NuGet.Configuration.PackageNamespacesConfiguration.GetConfiguredPackageSources(string term) -> System.Collections.Generic.IReadOnlyList<string>
-NuGet.Configuration.PackageNamespacesSourceItem
-NuGet.Configuration.PackageNamespacesSourceItem.Namespaces.get -> System.Collections.Generic.IList<NuGet.Configuration.NamespaceItem>
-NuGet.Configuration.PackageNamespacesSourceItem.PackageNamespacesSourceItem(string name, System.Collections.Generic.IEnumerable<NuGet.Configuration.NamespaceItem> namespaceItems) -> void
-NuGet.Configuration.PackageNamespacesSourceItem.SetKey(string value) -> void
+NuGet.Configuration.PackagePatternItem
+NuGet.Configuration.PackagePatternItem.PackagePatternItem(string pattern) -> void
+NuGet.Configuration.PackagePatternItem.Pattern.get -> string
+NuGet.Configuration.PackageSourceMapping
+NuGet.Configuration.PackageSourceMapping.GetConfiguredPackageSources(string term) -> System.Collections.Generic.IReadOnlyList<string>
```

Comment:
I think `packageId` makes more sense than `term`.

Reply:
Fixed in the latest commit.

Reply:
Actually mostly packagePrefix than packageId, but I'm ok with `packageId`.

Reply:
Isn't that API the package id exactly?
It's getting the sources for a specific package.

Reply:
Yes Nikolche that is correct, exact packageId is passed as parameter to this method to find out package sources associated with the longest matching prefix.